extension/diffheader.js

```diff
@@ -109,10 +107,9 @@ window.diffFileHeader = (() => {
 	const setup = () => {
 		$(window).on('scroll.diffheader', () => diffHeaderFilename());
 		const onResize = utils.debounce(() => diffHeaderFilename(true), 200);
-		$(window).on('resize', onResize);
+		$(window).on('resize.diffheader', onResize);
```

Comment:
[`EventEmitter2`](https://github.com/asyncly/EventEmitter2) has it. Would be sweet to see in core, though